README.md

```diff
@@ -1 +1,76 @@
-# modelplace
\ No newline at end of file
+This is an instruction on how to integrate a model to [Modelplace.AI](https://modelplace.ai/).  
+We support [PyTorch](https://pytorch.org/), [Tensorflow](https://www.tensorflow.org/), [OpenVINO](https://docs.openvinotoolkit.org/latest/index.html), and [OnnxRuntime](https://www.onnxruntime.ai/) backends. 
+
```

Comment:
It is not clear to me what steps need to be done to integrate a model into Modelplace
Please, add smth like this (not intend to copy/paste, feel free to change anything):

To provide a model we ask you to do the following:
1. **Provide information** about yourself and a model you want to publish
2. **Set up environment** to try our example
3. **Implement your model** in a similar with our example
4. **Pack your model** with setup.py

Reply:
Also, anchors should be used like:
[**Provide information**](#publishing-information) about yourself and a model you want to publish